tests/test_register_wallet.py

```diff
@@ -244,3 +219,32 @@ def test_register_wallet_not_sane_policy(client: Client):
 
     # TODO: we can probably not trigger stack and ops limits with the current limits we have on the
     # miniscript policy size; otherwise it would be worth to add tests for them, too.
+
+
+@has_automation("automations/register_wallet_accept.json")
+def test_register_unusual_singlesig_accounts(client: Client, speculos_globals):
```

Comment:
I don't see test with wsh, is it included in previous tests ?

Reply:
This test is for the typical single-signature accounts.
Most multisig and all the miniscript e2e tests use wsh, anyway.

(You could do a single signature account inside wsh, but it's pointless − mostly the goal here is to allow people to register accounts when they have funds locked in weird derivation paths).